app/models/webhooks/slack/base.rb

```diff
@@ -53,6 +63,14 @@ def discussion_link(text = nil, params = {})
     "<#{discussion_url(eventable.discussion, params)}|#{text || eventable.discussion.title}>"
   end
 
+  def vote_proposal_link(vote, position = nil)
+    vote_discussion_link position || vote.motion_name, { proposal: vote.proposal.key, position: vote.position }
+  end 
+
+  def vote_discussion_link(text = nil, params = {})
+    "<#{discussion_url(eventable.motion.discussion, params)}|#{text || eventable.motion.discussion.title}>"
```

Comment:
Hm.. I'd be curious if we could add

```
belongs_to :discussion, through: :motion
```
to `vote.rb`, and thus just be able to call `eventable.discussion` on the vote and get the discussion.

This should enable us to simply use `proposal_link` instead of needing to make a new set of methods.